tests/test_full_example_duckdb.py

```diff
@@ -108,6 +108,7 @@ def test_full_example_duckdb(tmp_path):
 
     linker_2 = DuckDBLinker(df, connection=":memory:")
     linker_2.load_settings(path)
+    linker_2.load_settings_from_json(path)
```

Comment:
Is this a mistake?

Reply:
I was on the fence about it. I left it in as it's such a lightweight test.

However, `load_settings_from_json` now just calls `load_settings`, so it's likely superfluous to requirements.